rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -61,16 +61,16 @@ msg_typename = '%s__%s__%s' % (spec.base_type.pkg_name, subfolder, spec.base_typ
 @[    if spec.base_type.pkg_name != field.type.pkg_name]@
 ROSIDL_GENERATOR_C_IMPORT
 @[    end if]@
-bool @(field.type.pkg_name)__@(lowercase_field_type)__convert_from_py(PyObject * _pymsg, void * _ros_message);
+bool @(field.type.pkg_name)__@(subfolder)__@(lowercase_field_type)__convert_from_py(PyObject * _pymsg, void * _ros_message);
 @[    if spec.base_type.pkg_name != field.type.pkg_name]@
 ROSIDL_GENERATOR_C_IMPORT
 @[    end if]@
-PyObject * @(field.type.pkg_name)__@(lowercase_field_type)__convert_to_py(void * raw_ros_message);
+PyObject * @(field.type.pkg_name)__@(subfolder)__@(lowercase_field_type)__convert_to_py(void * raw_ros_message);
```

Comment:
@apojomovsky I don't think this will be true in general e.g. a `_Request` message that is part of a service definition below `srv/` could have a field that's a message described in a message definition below msg/.

Reply:
You're right, I just restored this line. Thanks!